tests/gnrc_dhcpv6_client/Makefile

```
@@ -41,6 +41,8 @@ include $(RIOTBASE)/Makefile.include
 
 .PHONY: dhcpv6_server
 
+test-with-config/check-config: IFACE := ${IFACE}
```

```suggestion
test-with-config/check-config: export IFACE := ${IFACE}
```

this seems to be an export just for this rule

---

i did check that with an echo and an other similar variable
```
...
test-with-config/check-config: export IFACE := ${IFACE}
test-with-config/check-config: export IFACE2 := ${IFACE}

bob:test-with-config
        echo ${IFACE2}
...

```
 $ make bob
 ... working test...
 
echo 

 ```
 echo stays empty the rest works